### ISSUE TITLE
Cube is borken.

### ISSUE THREAD

**Issue by pdelia** [pdelia/away3d#208]
#28 Issue by __GoogleCodeExporter__, created on: 2015-04-24T07:51:13Z

```
What steps will reproduce the problem?
1. var cube:Cube = new Cube();
2.
3.

What is the expected output? What do you see instead?
TypeError: Error #1009: Cannot access a property or method of a null object
reference.
	at
away3d.primitives::Cube/buildCube()[/Users/Li/Development/Actionscript/Projects/
ITM3
Phase 2/Workspace/away3d_secure/src/away3d/primitives/Cube.as:100]

Please use labels and text to provide additional information.
The error occurs at the buildCube method of cube, line 100:
uv0 = makeUV(1-(face.uv0.u+offU), face.uv0.v+offV);


```

Original issue reported on code.google.com by `paleblue...@gmail.com` on 27 Jan 2009 at 4:33

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:13Z



Original comment by `jens...@gmail.com` on 3 Feb 2009 at 10:28
* Changed state: **Fixed**

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:13Z

```
This is fixed from 2.2.7
```

Original comment by `rob.bate...@gmail.com` on 4 Feb 2009 at 1:49
